[Sticky] Ability to set cooldown per channel.

cool-aid-man opened the pull request:
- Removed the class variable: REPOST_COOLDOWN
- Per channel configuration.
- Increased the default (minimum) cooldown from 3 to **5 seconds** (to respect the API rate limit).

cool-aid-man committed:
[Sticky] Ability to set cooldown per channel.

- Removed the class variable: REPOST_COOLDOWN
- Per channel configuration.
- Increased the default (minimum) cooldown from 3 to 5 seconds.

Tobotimus commented on sticky/sticky.py:
```diff
@@ -109,6 +108,19 @@ async def sticky_toggleheader(self, ctx: commands.Context, true_or_false: bool):
+    async def sticky_cooldown(self, ctx: commands.Context, seconds: int):
+        """Set the cooldown time for reposting sticky messages in this channel.
+        
+        The cooldown must be at least 5 seconds."""
+        if seconds < 5:
+            await ctx.send("The cooldown cannot be set lower than 5 seconds.")
```
I'd prefer to keep the previous default of 3 seconds
```suggestion
        The cooldown must be at least 3 seconds."""
        if seconds < 3:
            await ctx.send("The cooldown cannot be set lower than 3 seconds.")
```

Tobotimus commented on sticky/sticky.py:
```diff
@@ -30,6 +28,7 @@ def __init__(self, bot):
             header_enabled=True,
             advstickied={"content": None, "embed": {}},  # This is for [p]stickyexisting
             last=None,
+            cooldown=5,
```
```suggestion
            cooldown=3,
```

Tobotimus reviewed (approved):
Thanks so much for the contribution! cLGTM :heart:

cool-aid-man committed:
[Sticky] Set the default cooldown back to **3 seconds**

cool-aid-man commented:
Done toby!
Set the default cooldown back to old (i.e. **3 seconds**)

cool-aid-man committed:
[Sticky] Allows 'unsticky' to interrupt during cooldown.

- Fixed the issue where - the `unsticky` command wasn't invocable during cooldown (it used to wait till the cooldown is over then allow).
  - So, now, this allows 'unsticky' to be invoked during cooldown and re-instate the cooldown afterwards.
--------------
- Changed the `'cooldown'` command's behavior:
  - Now if nothing is passed then it'll set the channel-cooldown back to the DEFAULT_COOLDOWN value which is 3 seconds. Making the argument 'Optional'.
- Instead of hardcoding the default-cooldown into the 'cooldown' command, now it checks the `class constant` (DEFAULT_COOLDOWN = 3)
--------------------
- Added a `class constant` (    DEFAULT_COOLDOWN = 3) to maintain the consistency.
